notebooks/daniel_notebook.ipynb

```diff
@@ -1,191 +1,26 @@
 {
```

Comment:
<p>Here, present a little background on Newton's second law. Describe how forces are vectors and moments are vectors that equal r x F.</p><p> </p><p>Final static result is sum(F) = 0 and sum(M) = 0</p><p> </p><p>In this example problem, make sure you introduce:</p><p> </p><p>there are 2 applied forces at a point and 1 unknown force. Here, you sum the known forces and solve for the 3 components that define the unknown force. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cooperrc/uconn_numpy_tutorials/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>